cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client_test.go

```diff
@@ -41,6 +42,10 @@ func TestWaitForOp(t *testing.T) {
 	server := test_util.NewHttpServerMock()
 	defer server.Close()
 	g := newTestAutoscalingGceClient(t, "project1", server.URL)
+
+	g.operationPollInterval = 5 * time.Millisecond
```

Comment:
Is there any reason for this not to be 0?

Reply:
Done